src/output-json-frame.c

```diff
@@ -190,12 +191,12 @@ static void FrameAddPayloadUDP(JsonBuilder *js, const Packet *p, const Frame *fr
 
     uint32_t frame_len;
     if (frame->len == -1) {
-        frame_len = p->payload_len - frame->offset;
+        frame_len = p->payload_len - (uint32_t)frame->offset;
```

Comment:
Packet `payload_len` is defined as `uint16_t` and Frame `offset` as `uint64_t`. How do we justify this?

Reply:
Right we can cast `frame->offset` to u16 as it is previously checked smaller than `payload_len`